test.sh

```diff
@@ -4,81 +4,81 @@ echo "* Test #1"
 echo "For alignment with large deletion and micro insersion. Mismatches"
 echo "and small deletions are present in each flanking sequence."
 echo "Run as: ./age_align test1_*.fa"
-./age_align test1_*.fa
+./age_align test1_*.fa "$@"
```

Comment:
This allows us to use the test cases with and without extra flags, e.g. the new `-allpos` flag.